src/data/matrix/basic.lean

```diff
@@ -60,6 +60,11 @@ by { ext, simp, }
 def transpose (M : matrix m n α) : matrix n m α
 | x y := M y x
 
+/-- The conjugate transpose of a matrix. -/
+def conj_transpose [has_star α] (M : matrix m n α) : matrix n m α
+| x y := star (M y x)
+localized "postfix `ᴴ`:1500 := matrix.conj_transpose" in matrix
```

Comment:
I think it would make sense to put this on line 885 instead, just before the `section star_ring`

Reply:
Sure. I will also update the other PR. If I committed the change here, how can I update my local version? @eric-wieser

Reply:
I recommend you pause work entirely on your other PR (#8289), and focus on just this one until it gets merged; so just make the changes on your `conj_trans_lz` branch.

Reply:
Actually, I take back my comment - this probably is a good place to have this. I'll make another comment thread.

Reply:
I mean update the local version for this PR @eric-wieser

Reply:
perhaps there is no further issue with this one

Reply:
Oh, you mean how do you update your local version to get the changes I suggested on github? Just run `git pull`.